internal/indexer/position_processor.go

```
@@ -0,0 +1,184 @@
+			"entry_method":          entryMethod,
+			"exit_method":           nil,
+			"is_terminated":         false,
+			"neutron_address":       nil,
+		}
+
+		_, _, err = p.db.From("positions").Insert(positionRecord, false, "", "", "").Execute()
+		if err != nil {
+			return fmt.Errorf("failed to create new position: %w", err)
+		}
+
+		log.Printf("Created new position for account %s: amount = %s, index = %d", positionRecord["ethereum_address"], positionRecord["amount"])
```

golangci-lint (1.64.8)

printf: log.Printf format %d reads arg #3, but call has 2 args

(govet)